### PR TITLE
proposed changes to stage volunteer roles

### PR DESCRIPTION
**Pull request by blip2**
proposed changes to the volunteer roles. for review by wider stage team. happy for any comments from volunteer team too!

### CODE REVIEW

File: apps/volunteer/data/roles/bands_lighting_operator.yml [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
-  **You'll need to attend one of our training sessions before your shift, these
-  will be run daily, and are a great opportunity to learn more about how lighting
-  for live performances works.**
+  We encourage mentoring of other volunteers who may be keen to learn, helping 
+  them to gain knoweldge and experience of operating a lighting desk.
+
+  **This role requires experience with Avolites lighting desks. If you're not 
+  confident busking large moving light rigs please consider taking the stage crew
+  role adn we will try and give you a chance to operate the desk..**
```
Comment (lukegb) [inline apps/volunteer/data/roles/bands_lighting_operator.yml:13]: typo: and

File: apps/volunteer/data/roles/bands_sound_operator.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 full_description_md: >-
-  As a sound operator you'll be operating the Yamaha DM7 sound desk for bands
+  As the sound operator you'll be operating our Yamaha DM7 sound desk for bands
   and live performances, doing sound checks, making sure the bands and performers
   are happy, and that everything sounds great for the audience.
 
-  You'll also be mentoring volunteer sound technicians, helping them to gain
-  experience operating a sound desk.
+  You'll also be mentoring a volunteer sound technicians, helping them to gain
+  knoweldge and experience of operating a sound desk.
```
Comment (lukegb) [inline apps/volunteer/data/roles/bands_sound_operator.yml:9]: typo: knowledge

File: apps/volunteer/data/roles/bands_sound_operator.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 description: "Operating the sound desk for live performances"
 full_description_md: >-
-  As a sound operator you'll be operating the Yamaha DM7 sound desk for bands
+  As the sound operator you'll be operating our Yamaha DM7 sound desk for bands
   and live performances, doing sound checks, making sure the bands and performers
   are happy, and that everything sounds great for the audience.
 
-  You'll also be mentoring volunteer sound technicians, helping them to gain
-  experience operating a sound desk.
+  You'll also be mentoring a volunteer sound technicians, helping them to gain
```
Comment (lukegb) [inline apps/volunteer/data/roles/bands_sound_operator.yml:8]: either drop the a again or depluralise technicians

**Comment by jellybob** [comment]
Thanks! I've pulled those into the upstream branch as there were some changes to YAML formatting which mean these will conflict.

**Pull request closed without merging**